docs/source/modin/supported/index_supported.rst

```diff
@@ -109,7 +109,8 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``max``                     | N                               |                                  |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``reindex``                 | N                               |                                  |                                                    |
+| ``reindex``                 | P                               |                                  | ``N`` if the Index is MultiIndex and lazy, or      |
```

Comment:
Should we also add a note that while pandas will raise an error for reindexing an index with duplicate values, Snowpark pandas will not and will instead do a left join between the two indexes?

Reply:
I added that note to the docstring - usually behavior change notes go in the docstring?